examples/player/simid_player.js

```diff
@@ -9,8 +9,9 @@ class SimidPlayer {
    * Sets up the creative iframe and starts listening for messages
    * from the creative.
    * @param {!Function} This function gets called when the ad stops.
+   * @param {boolean} Represents if the ad is a linear one
```

Comment:
Format for these should be 
`@param {type} variableName Description.`

Make sure you add the variable name in and end with a period